src/problem_bank_scripts/problem_bank_scripts.py

```diff
@@ -690,6 +690,27 @@ def process_file_upload(part_name, parsed_question, data_dict):
     return replace_tags(html)
 
 
+def process_code_input(part_name, parsed_question, data_dict):
```

Comment:
Where possible, I try to use the exact same name as the element, so suggest changing this to `process_file_editor()`